### ISSUE TITLE
chapter2: PyTorch快速入门 代码出现错误

### ISSUE THREAD

**Issue by kevinyu1949** [chenyuntc/pytorch-book#101]
在**CIFAR-10数据加载及预处理**这一块的代码，**In[43]** :

原代码：
`trainloader = t.utils.data.DataLoader(trainset, batch_size=4,shuffle=True, num_workers=2)`

`testloader = t.utils.data.DataLoader(testset,batch_size=4, shuffle=False,num_workers=2)`

运行时，应改为：
> **t.utils.data.DataLoader( )  -->  torch.utils.data.DataLoader( )**
**即 ： t --> torch**
